contracts/mainnet/IPolymorphV1.sol

```diff
@@ -3,9 +3,6 @@ pragma solidity ^0.7.0;
 
 import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
```

Comment:
Yes I think we can delete IPolymorphV1.sol, as we have both functions exposed in Polymorph.sol